headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/AttachmentResourceImpl.java

```diff
@@ -416,7 +416,9 @@ private Attachment _updateAttachment(
 				GetterUtil.get(
 					attachment.getPriority(),
 					cpAttachmentFileEntry.getPriority()),
-				attachment.getType(), serviceContext);
+				GetterUtil.get(
```

Comment:
@andrea-ale-sbarra , we generally shouldn't use `GetterUtil.get`, but instead use `GetterUtil.getString` (or whatever type this actually is). This makes it a lot more obvious what type the variable is that's getting passed in. Can you send a change for all the `GetterUtil.get` instances in this `_cpAttachmentFileEntryService.updateCPAttachmentFileEntry` method call?